### ISSUE TITLE
DOC: stats: multivariate distribution documentation issues

### ISSUE THREAD

**Issue by mdhaber** [scipy/scipy#12848]
As noted in gh-12839, there are a few issues that affect the documentation of several multivariate distributions.

- [x] Class documentation defines a parameter `x`, but `x` is not a parameter of the initializer.

![image](https://user-images.githubusercontent.com/6570539/93006843-22ca7280-f516-11ea-832b-1c2e16599710.png)

- [x] Class documentation defines a parameter `random_state` instead of `seed`.

![image](https://user-images.githubusercontent.com/6570539/93006871-8654a000-f516-11ea-9f05-031af2240e5c.png)

![image](https://user-images.githubusercontent.com/6570539/93006860-5c02e280-f516-11ea-9ad8-4758ab3660a4.png)

- [x] The text in the class documentation about returning a frozen distribution appears in bold like parameter names, and there is an unnecessary bullet point before "Frozen"

![image](https://user-images.githubusercontent.com/6570539/93006922-24486a80-f517-11ea-81f5-a113fad6829d.png)

- [x] The Methods sections renders with double backticks around each method signature.

![image](https://user-images.githubusercontent.com/6570539/93006880-a5533200-f516-11ea-8c5f-c782724d64f5.png)

- [ ] Some parameters of the methods (e.g. `size`, `abseps`, `releps`) are not defined

![image](https://user-images.githubusercontent.com/6570539/93006895-d9c6ee00-f516-11ea-8397-9252eb384587.png)

- [ ] Related to the problems above, and true of univariate distributions, too: method names do not link to any documentation. Perhaps this is intended, but I think it would be useful to at least link to the `rv_continuous` method documentation. In any case, it looks like the author of `multinomial` intended for the documentation to be rendered somewhere, e.g.

```
    def logpmf(self, x, n, p):
        """
        Log of the Multinomial probability mass function.

        Parameters
        ----------
        x : array_like
            Quantiles, with the last axis of `x` denoting the components.
        %(_doc_default_callparams)s

        Returns
        -------
        logpmf : ndarray or scalar
            Log of the probability mass function evaluated at `x`

        Notes
        -----
        %(_doc_callparams_note)s
        """
```

Also, not sure how closely related gh-11578 is, but it is worth mentioning here.

**Comment by WarrenWeckesser** [comment]
The problem with the double backticks has an easy fix: remove the backticks.  Sphinx will render those lines correctly without them.

We might want to hold off fixing that until https://github.com/scipy/scipy/pull/12839 is merged.

**Comment by tirthasheshpatel** [comment]
While trying to address this issue, I found a few others:

- [ ] `Notes` are not rendered properly for some multivariate distributions.
![image](https://user-images.githubusercontent.com/43181252/99872533-53124980-2c08-11eb-9155-440f7e849221.png)

- [ ] `ortho_group` (multivariate) distribution doesn't have a frozen distribution class.

- [ ] `matrix_normal` distribution has parameter `X` (capital x) in its `pdf` and `logpdf` methods which is inconsistent with other distributions (that have a lower case `x` instead).

**Comment by tirthasheshpatel** [comment]
Also, @mdhaber, the 5'th point says:

> Some parameters of the methods (e.g. size, abseps, releps) are not defined

These parameters are defined in the methods that have them but sadly the documentation page doesn't link to the documentation of these methods. Perhaps, this should be merged with the 6'th point about linking the documentation of methods.

**Comment by rajpratyush** [comment]
@mdhaber is this issue open for work??

**Comment by mdhaber** [comment]
This is not a very simple issue.  It's partially held up by [this issue](https://github.com/scipy/scipy/issues/10328#issuecomment-770108887), which is a problem with all distributions. I am working on other things right now, though, so I can't really help on that at this time.

**Comment by mdhaber** [comment]
Hmm, well @tirthasheshpatel had an open PR, gh-13103. We put that on hold because we ran into difficulty and did not find others who were available to help. So not really, I'm afraid.

**Comment by rajpratyush** [comment]
can i try?

**Comment by tirthasheshpatel** [comment]
Hi @rajpratyush! I agree with @mdhaber. You can follow the discussion in gh-13103 to get a bit of context to work on this issue. If you can get to link the documentation of methods in the class documentation or separate the documentation of the class parameters and method parameters, it would be very welcome. I can help review and verify the approach but I am not too familiar with ``sphinx`` and ``numpydoc`` right now to do that myself. From a very cursory look, it seems that it would be a lot of work. So, if you are up for a challenge, do propose a PR. You will find some maintainers who can help once you propose a PR. Hope this helps!

**Comment by tirthasheshpatel** [comment]
> I am familiar with `sphinx` but not with `numpydoc` Could anyone post here any reference material for the same ?

You can read the [documentation](https://numpydoc.readthedocs.io/en/latest/). A very nice and comprehensive example can be found [here](https://numpydoc.readthedocs.io/en/latest/example.html).
